src/controllers/main/main.ts

```diff
@@ -333,6 +344,98 @@ export class MainController extends EventEmitter {
     this.emitUpdate()
   }
 
+  async importSmartAccountFromDefaultSeed(seed?: string) {
+    await this.withStatus(
+      'importSmartAccountFromDefaultSeed',
```

Comment:
Minor: you could you use our pattern with `#importSmartAccountFromDefaultSeed`, to avoid 1 level of nesting, but it's fine as is also.